.env.example

```diff
@@ -59,3 +59,16 @@ VITE_PUSHER_HOST="${PUSHER_HOST}"
 VITE_PUSHER_PORT="${PUSHER_PORT}"
 VITE_PUSHER_SCHEME="${PUSHER_SCHEME}"
 VITE_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
+
+# social accounts
+GITHUB_CLIENT_ID=
```

Comment:
@nikoladomazetovikj Please provide info/readme on how this is integrated and to be used.